asset_manager/storage.py

```diff
@@ -77,85 +73,32 @@ def read_string_from_object(
     -------
     string:
         The string read from the object.
-    '''
+    """
     bucket = _bucket_from_argument(bucket)
     b = BytesIO()
-    bucket.download_fileobj(Key=object_name, Fileobj=b)
+    try:
+        bucket.download_fileobj(Key=object_name, Fileobj=b)
+    except NoCredentialsError as exc:
+        raise Exception("No s3 credentials, you may need to sign in") from exc
     # Must return to the beginning of the file before reading.
     b.seek(0)
     result_bytes = b.read()
     result_str = result_bytes.decode()
     return result_str
 
 
-def list_objects_in_bucket(
-    bucket: Any = None
-) -> List[str]:
-    '''
+def list_objects_in_bucket(bucket: Any = None) -> List[str]:
+    """
     Get the names of all objects in a bucket.
 
     Parameters
     ----------
     bucket:
         An S3 bucket object. Defaults to the bucket in the configuration file.
-    '''
+    """
     bucket = _bucket_from_argument(bucket)
-    objects = [o.key for o in bucket.objects.all()]
+    try:
+        objects = [o.key for o in bucket.objects.all()]
+    except NoCredentialsError as exc:
+        raise Exception("No s3 credentials, you may need to sign in") from exc
     return objects
-
-
-@overload
-def get_secret(
-    secret_name: str,
-    assume_json: Literal[True] = True
-) -> Dict[str, str]: ...
-
-
-@overload
-def get_secret(
-    secret_name: str,
-    assume_json: Literal[False]
-) -> Union[str, bytes]: ...
-
-
-def get_secret(secret_name: str, assume_json: bool = True):
```

Comment:
I assume there was a time when I was (misguidedly) using Secrets Manager to hold a credential to my S3 bucket. Ah, young and naive. Now this code is completely unused.